src/nvim/api/ui.c

```diff
@@ -305,6 +305,21 @@ bool remote_ui_restart(uint64_t channel_id, Error *err)
   return true;
 }
 
+// Returns true if the 'connect' event was successfully sent to the UI
+void remote_ui_connect(uint64_t channel_id, char *server_address, Error *err)
+{
+  RemoteUI *ui = get_ui_or_err(channel_id, err);
+  if (!ui) {
+    emsg(err->msg);
+    return;
+  }
```

Comment:
Why is the error message printed here when there is already proper handling in `ex_connect()`?